### DISCUSSION TITLE
Assigning specific GPU to use for NVENC.HEVC

### DISCUSSION THREAD

**Discussion by bricor1709** [RandomEngy/VidCoder#996] [Q&A] [answered]
Have dual Discrete GPUs in my system RTX 3070 ti and Quadro P4000.
Want to assign P4000 to be used exclusively for remux/rerender work in Vidcoder (NVENC 265 Hevc) while I am playing games on the 3070. No monitor on P4000
I cannot find any way to assign the P4000, is there even a way? 
GPU 0 GPU 1 or something and where would it be in Vidcoder?

**Comment by RandomEngy** [comment]
There isn't a way to do that right now. The HandBrake API doesn't expose a mechanism to do so.

**Reply by bricor1709** [reply]
So the way that Handbrake do it (https://handbrake.fr/docs/en/latest/technical/video-nvenc.htm)
is not exposed in the API? I thought Vidcoder was completely based on Handbrake

**Comment by bricor1709** [comment, marked as answer]
Seems that it works. just adding gpu=0 or 1,2,3
in Encoding settings > Video Encoding > More Settings, worked for me.

**Reply by RandomEngy** [reply]
My bad! VidCoder just passes through all of those advanced options without needing to be aware of them. Glad you found the way to make it work!
